basic-examples/quoting/README.md

```diff
@@ -1,15 +1,16 @@
-# Expressions
+# Quoting
 
 ## Workflow Description
 
-This workflow demonstrates example expressions as detailed in our [expression documentation](https://arcalot.io/arcaflow/workflows/expressions/).
+This workflow demonstrates example expressions with quotes as detailed in our [expression documentation](https://arcalot.io/arcaflow/workflows/expressions/).
+There are multiple types of quoting, and quotes must pass through YAML's syntax. Look at this example to learn about the various ways to determine what would work best for your use case.
 
 ## Files
 
 - [`workflow.yaml`](workflow.yaml) -- Defines the workflow input schema, the plugins to run
-  and their data relationships, and the output to present to the user
+  and their data relationships, and the output to present to the user.

```

Comment:
If we're adding more than the general, boilerplate description (i.e., you've mentioned that the input is empty, and I've proposed a specific description for the subworkflow), I think it would be good to include a synopsis of what _this_ `workflow.yaml` includes, e.g.:
> For each string in a list of inputs, this workflow invokes a subworkflow with the string as its input; this workflow then outputs a list of the values returned from each invocation of the subworkflow.

Reply:
Changed.